src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services_with_short_service_closure.yml

```diff
@@ -0,0 +1,9 @@
+
+services:
+    service_container:
+        class: Symfony\Component\DependencyInjection\ContainerInterface
+        public: true
+        synthetic: true
+    foo:
+        class: Foo
+        arguments: ['@>?bar']
```

Comment:
Why this way and not this other? I'm asking because none of them looks more correct to me. It's just a sequence of characters with very special meanings.
```yaml
        arguments: ['@?>bar']
```

Reply:
It could go either way. It's easier to say ? sticks to the argument and means optional and @> means service closure -- as a way to remember it resembles `fn () =>` quite a bit.

Searching the docs should be no harder than searching for @@ or @?

Reply:
Let's keep `@>?` for "service", "as closure", "ignore invalid reference".

Reply:
That's exactly what this is